demos/basic.demo

```diff
@@ -0,0 +1,34 @@
+# Demo: how to install, configure and use Knvavigator in a local k8s cluster, such as minikube or KinD
+
+# Show the cluster infomation
+kubectl cluster-info
+
+kubectl get nodes
+
+# Intall KWOK node simulator 
+
+KWOK_REPO=kubernetes-sigs/kwok
+KWOK_LATEST_RELEASE="v0.5.2"
+kubectl apply -f "https://github.com/${KWOK_REPO}/releases/download/${KWOK_LATEST_RELEASE}/kwok.yaml"
+kubectl apply -f "https://github.com/${KWOK_REPO}/releases/download/${KWOK_LATEST_RELEASE}/stage-fast.yaml"
+kubectl apply -f https://github.com/${KWOK_REPO}/raw/main/kustomize/stage/pod/chaos/pod-init-container-running-failed.yaml
+kubectl apply -f https://github.com/${KWOK_REPO}/raw/main/kustomize/stage/pod/chaos/pod-container-running-failed.yaml
+
+# Add virtual nodes to the cluster
+helm install virtual-nodes charts/virtual-nodes -f charts/virtual-nodes/values.yaml
+
```

Comment:
Can add "kubectl get node" after virtual nodes are installed, so we can show them?

Reply:
I've added it.

Reply:
I've add the demo to README. Take a look please. https://github.com/NVIDIA/knavigator/blob/db654569d08dc1f869b090a41a7755e053c628d7/README.md

Reply:
BTW, I've created a similar demo for the DRA project too.  https://github.com/NVIDIA/k8s-dra-driver